README.md

```diff
@@ -1,12 +1,16 @@
 # Cohort Manager
 
-```
-_________        .__                   __       _____                                             
-\_   ___ \  ____ |  |__   ____________/  |_    /     \ _____    ____ _____     ____   ___________ 
-/    \  \/ /  _ \|  |  \ /  _ \_  __ \   __\  /  \ /  \\__  \  /    \\__  \   / ___\_/ __ \_  __ \
-\     \___(  <_> )   Y  (  <_> )  | \/|  |   /    Y    \/ __ \|   |  \/ __ \_/ /_/  >  ___/|  | \/
- \______  /\____/|___|  /\____/|__|   |__|   \____|__  (____  /___|  (____  /\___  / \___  >__|   
-        \/            \/                             \/     \/     \/     \//_____/      \/       
+```txt
```

Comment:
I mean I respect the effort, definitely like it

Reply:
Critical change if you ask me. 🧑‍🎨